### PR TITLE
6/build website

### LINKED ISSUE #6: Build website
Build out pages and add content.

### PR DESCRIPTION
**Pull request by AdamVig**
Closes #6.

### CODE REVIEW

**Comment by AdamVig** [comment]
*Note: Screenshots are for reference only! They are not a very good representation of the website.*
# Homepage
image
image
image


# Subscribe Page
image

File: src/index.html [hunk 1]
```diff
@@ -3,33 +3,104 @@
...
+                        find exactly the item you need, when you need it.
+                    </p>
+                </div>
+                <div class="col-xs-6 col-md-3">
+                    <div class="stockpile-inventory"></div>
+                </div>
+            </section>
+            <section class="row center-xs middle-md">
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <div class="stockpile-inventory"></div>
```
Comment (emroussel) [inline src/index.html]: Should this be `<div class="stockpile-scan"></div>` instead?

File: src/index.html [hunk 2]
```diff
@@ -3,33 +3,104 @@
...
+                    <p>
+                        To return items, simply scan them again.
+                    </p>
+                </div>
+            </section>
+            <section class="row center-xs middle-md">
+                <div class="col-md-offset-2 col-xs-6 col-md-3">
+                    <p>
+                        Create kits of items that go together so you don't forget
+                        anything when you rent items.
```
Comment (emroussel) [inline src/index.html]: I think reading through the different sections, the word 'item' is repeated a lot. Is there a way we can rephrase some of them to make it flow a little better?

**Review by emroussel — CHANGES_REQUESTED** [review]
Great job!

I think in general I would add more space between every sections and between the header and the first section.

Also, I'm not sure if this was voluntary or not, but the website as a whole feels off centered. The sections in the middle are aligned to the right. There is also more space between the text and the image in the section for scanning than there is for the section about viewing inventory and kits.

Finally, I feel like screenshots would look better if they are placed inside a phone frame.

**Comment by AdamVig** [comment]
Awesome feedback, thanks! I realized as soon as I looked at it again that it needed a lot more work. 

I think I addressed each of your pieces of feedback. If I missed anything, please tell me!

Here are updated screenshots, which are actually accurate this time (I used an extension instead of hand-capturing them 😆).

### Laptop with HiDPI Screen
image

### iPad
image

### iPhone 6
image

**Comment by emroussel** [comment]
Yes it looks much better, good job!

Two more things that I thought of when looking at it again:
- Maybe adding a title for each section would be helpful to indicate which image/screenshot is associated with what text on mobile (One or very few words describing what the section is about). The whitespace helps, but I still had a few moments of hesitation when I first saw it. It could be above the text or centered above both the image and text, not sure which would look better. Could be similar to [stripe.com](https://stripe.com/) if you think it fits for Stockpile.
- If you had titles for sections, it might make sense to have a navigation bar in the header that has links to these different sections.

Also, I just realized that we never mention that what we are selling is an app 😂

Might be worth adding that to the description or somewhere else. Another option might be to have the classic two images of iOS and Android with our app instead of where the logo is right now and move the logo to the navbar or something.

**Comment by AdamVig** [comment]
### 1080p Screen
image

### iPhone 6
image

I decided to pass on adding section links to the nav bar. The page is so short and simple that I don't think section links would add any value. I will add links to the subscribe page and the barcodes page once they exist, but that will be in each of their pull requests, not this one.

I thought about how to add the word "app" in somewhere, but couldn't see any place where it fit naturally. I think the screenshots do a good enough job of getting the point across. We'll have to do some user testing of our website copy, though, to check if it actually communicates well.

**Review by emroussel — APPROVED** [review]

**Comment by emroussel** [comment]
Sounds good, and I love the titles of sections!

**Pull request merged**